packages/react-composites/src/composites/CallComposite/styles/NoticePage.styles.ts

```diff
@@ -3,24 +3,31 @@
 
 import { IStyle, IStackTokens } from '@fluentui/react';
 
+/**
+ * @private
+ */
+export const containerStyle: IStyle = {
+  width: '22.5rem'
```

Comment:
call composite min width will be 19.5rem for mobile and 30rem for desktop, so this should be a max-width.
Wondering how this look on iPhone SE (screen width of 20rem)